statler_api/models/review.py

```diff
@@ -13,6 +13,11 @@ class Review(models.Model, StatlerModel):
     text = models.TextField()
     rating = models.FloatField()
     timestamp = models.DateTimeField()
+    top_review_rank = models.IntegerField(null=True, blank=True, default=None)
+
+    class Meta:
+        # Disallow multiple "equally top" reviews on the same play.
+        unique_together = ("play", "top_review_rank")
```

Comment:
This assumes that top reviews are ordered (best review, second best review, etc.), not simply "top or not". We should decide if that's what we want.